backend/app/core/background_runner.py

```diff
@@ -0,0 +1,54 @@
+import asyncio
+import json
+
+from app.core.bot import BaseBotClient
+from app.core.config import settings
+
+
+class BackgroundRunner:
```

Comment:
I would make this class name specific to the fact it's a background runner for the matrix bot, rather than other long running tasks we might need?

Reply:
Done